### PR TITLE
update terraform docs version

### PR DESCRIPTION
**Pull request by umairidris**

### CODE REVIEW

File: infra/build/Makefile [hunk 1]
```diff
@@ -16,7 +16,7 @@ SHELL := /usr/bin/env bash # Make will use bash instead of sh
 TERRAFORM_VERSION := 0.13.4
 CLOUD_SDK_VERSION := 313.0.1
 GSUITE_PROVIDER_VERSION := 0.1.22
-TERRAFORM_DOCS_VERSION := 0.6.0
+TERRAFORM_DOCS_VERSION := 0.10.1
```
Comment (bharathkkb) [inline infra/build/Makefile:19]: We will also need to bump https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/1967f1842600a982b2425f114388e76db6403c88/infra/build/Makefile#L30

**Review by bharathkkb — COMMENTED** [review]
@morgante can we cross port this for the 0.12 image and release an image with `0.12` `0.12.1` tags?

**Comment by morgante** [comment]
> @morgante can we cross port this for the 0.12 image and release an image with 0.12 0.12.1 tags?

Yeah you'll just need to cherry-pick and cut the release manually.

**Comment by bharathkkb** [comment]
Tried a local build with `0.10.1` yesterday and it looks like it wont work out of the box with[ our wrapper](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/build/developer-tools/build/scripts/task_wrapper_scripts/terraform_docs.sh) which has hacks for 0.12 (which is now natively supported). I will work on modifying the terraform_docs.sh script to make it compatible with 0.10.1

**Comment by bharathkkb** [comment]
superseded by #815

**Pull request closed without merging**